cryostatio/cryostat-helm
# feat(truststore): declarative configuration of TLS trusted certificate Secrets

See also #215
Fixes #217

To test:
1. Get a TLS certificate. I will use https://github.com/andrewazores/vertx-fib-demo/blob/master/src/main/extras/app/resources/vertx-fib-demo.cer as an example.
2. Create a Secret from the cert: `oc create secret generic fib-tls --from-file=./vertx-fib-demo.cer`
3. Install chart with this cert trusted: `helm install --set core.config.tlsTruststore.secretNames='{fib-tls}'`
4. Wait for Cryostat Pod to be Ready, then check its logs. Look for this:
```
Importing certificate /truststore/fib-tls/..2024_11_27_20_47_48.3800988412/vertx-fib-demo.cer ...
Certificate was added to keystore
```
